azurerm/helpers/azure/app_service.go

```diff
@@ -207,11 +231,63 @@ func SchemaAppServiceSiteConfig() *schema.Schema {
 					Type:     schema.TypeString,
 					Optional: true,
 				},
+
+				"cors": SchemaAppServiceCorsSettings(),
```

Comment:
since this schema is also shared with the App Service Slot resource, could we add the same acceptance test / docs for that resource too?